tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+###############################################################################
```

Comment:
what's the meaning of 'conftest'?

Reply:
`conftest.py` is the way that `pytest` shares fixtures across the entire test suite.
https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions